app/controllers/appeals_controller.rb

```
@@ -34,12 +34,14 @@ def show_case_list
     end
   end
 
-  def document_count
-    render json: { document_count: appeal.number_of_documents }
-  rescue Caseflow::Error::EfolderAccessForbidden => e
-    render(e.serialize_response)
-  rescue StandardError => e
-    handle_non_critical_error("document_count", e)
+  def document_counts_by_id
+    render json: {
```

Let's raise an error if there are more than 5 appeal IDs in the request. We don't want to spin up more than 5 threads per request